isaric/parsers/ncov-phosp.toml

```diff
@@ -0,0 +1,803 @@
+    { field = "crf1a_com_mer_other", if = { crf1a_com_mer_other = { "!=" = "NA" } } },
+    { field = "crf1a_com_mh_other", if = { crf1a_com_mh_other = { "!=" = "NA" } } },
+  ]
+
+[subject.has_died] # type - boolean
+  field = "crf2a_reason_not_done"
+  values = { Language = false, Declined = false, Died = true, Other = false }
+
+  # VISIT
+  # required fields - "visit_id", "subject_id", "country_iso3", "start_date", "outcome", "date_outcome"
+
+[visit]
+  country_iso3 = "GBR"
+  outcome = "discharged"
+
+  [visit.visit_id]
+    field = "phosp_id"
+    sensitive = true
+
+  [visit.subject_id]
+    field = "phosp_id"
+    sensitive = true
+
+  [visit.start_date]
+    field = "crf1a_date_adm"
+    description = "Date of admission to this hospital"
+    if = { crf1a_date_adm = { "!=" = "NA" } }
+
+  [visit.transfer_from_other_facility]
+    field = "crf1a_xfer_in"
+    ref = "Y/N"
+
+  [visit.treatment_dialysis]
+    combinedType = "any"
+    fields = [
+      { field = "crf1a_add_diag_cvvh", description = "b) Renal failure requiring CVVH or Haemodialysis", ref = "Y/N" },
+      { field = "crf1a_treat_rrt", ref = "Y/N" },
+    ]
+
+  [visit.treatment_ecmo]
+    field = "crf1a_o2_ecmo"
+    ref = "Y/N"
+
+  [visit.treatment_corticosteroid]
+    field = "crf1a_treat_ss"
+    ref = "Y/N"
+
+  [visit.treatment_oxygen_therapy]
+    combinedType = "any"
+    fields = [
+      { field = "crf1a_o2_supp", description = "a) Supplemental oxygen", ref = "Y/N" },
+      { field = "crf1a_supp_o2", description = "Supplemental oxygen", ref = "Y/N" },
+      { field = "crf1a_o2_hfn", ref = "Y/N" },
+      { field = "crf1a_o2_imv", ref = "Y/N" },
+      { field = "crf1a_o2_ecmo", ref = "Y/N" },
+      { field = "crf1a_o2_blniv", ref = "Y/N" },
+      { field = "crf1a_o2_cpapv", ref = "Y/N" },
+    ]
+
+  [visit.treatment_invasive_ventilation]
+    field = "crf1a_o2_imv"
+    description = "e) Invasive Mechanical Ventilation"
+    ref = "Y/N"
+
+  [visit.treatment_antibiotics]
+    field = "crf1a_treat_at"
+    description = "a) Antibiotic therapy"
+    ref = "Y/N"
+
+  [visit.treatment_noninvasive_ventilation]
+    field = "crf1a_o2_blniv"
+    description = "c) Bi-Level Non-Invasive Ventilation"
+    ref = "Y/N"
+
+  [visit.treatment_high_flow_nasal_cannula]
+    field = "crf1a_o2_hfn"
+    description = "d) High Flow Nasal O2"
+    ref = "Y/N"
+
+  [visit.treatment_immunosuppressant]
+    combinedType = "any"
+    fields = [
+      { field = "crf1a_treat_saril", ref = "Y/N" },
+      { field = "crf1a_treat_tocil", ref = "Y/N" },
+    ]
+
+  [visit.treatment_prone_position]
+    field = "crf1a_treat_proning"
+    ref = "Y/N"
+
+  [visit.treatment_anticoagulation]
+    field = "crf1a_treat_tdac"
+    ref = "Y/N"
+
+  [visit.treatment_cardiovascular_support]
+    field = "crf1a_o2_ecmo"
+    ref = "Y/N"
+
+  [visit.treatment_monoclonal_antibody]
+    field = "crf1a_treat_saril"
```

Comment:
@ekamau this might need some clinical input if possible - sarilumab falls under ATC class immunosuppressants and is mapped in the treatment_immunosuppressant attribute. It is also described as a monoclonal antibody treatment by BNF https://bnf.nice.org.uk/drugs/sarilumab/. So is it ok to include in both classes?

Reply:
I think for now we can leave it in both classes - immunosuppressant and monoclonal antibody ..